src/igvfd/schemas/changelogs/rodent_donor.md

```diff
@@ -3,3 +3,4 @@
 ### Minor changes since version 1
```

Comment:
Can add an extra blank line after this header to match the changes in the other changelogs

Reply:
fixed